### DISCUSSION TITLE
validation

### DISCUSSION THREAD

**Discussion by barupal** [idslme/IDSL.SpectraPrediction#2] [Announcements]
## 👋 Validation

We created a validation set of compounds from the Blood Exposome Database (www.bloodexposome.org),  that were present in the NIST23 database, but not in the training set (NIST20+GNPS+MoNA). We validated it only for the M+H ESI adduct. 

The set has ~2000 unique InchiKey14 (2D structures). For this validation set - a median dot-product of 620 using NIST MS Search method was achieved, suggesting that ICEBERG Model framework ( https://github.com/samgoldman97/ms-pred ) can potentially predict useful CID MS/MS spectra for a majority of the compounds in the blood exposome database (www.bloodexposome.org).

**Comment by barupal** [comment]
It seems the model accuracy is higher for compounds with lower exact mass because the training-set is slightly skewed towards lower mass range. There seems to be also a correlation between A (the Tanimoto Coefficient between x compound in the validation set and the most similar compound in the training data) and B (dot-product between the simulated spectra and NIST23 spectra for compound x).  
![spectra_prediction_validation_set_plots](https://github.com/user-attachments/assets/2b7db009-30e0-4372-8067-89b8f8330ee3)
